config/locales/de.yml

```diff
@@ -1,124 +1,149 @@
+---
 de:
-  time:
-    formats:
-      default: "%d.%m.%y %H:%M"
-  helpers:
-    submit:
-      create: '%{model} erstellen'
-      update: '%{model} aktualisieren'
-    label:
-      question_group:
-        survey: Umfrage
-        questions: Umfragen
-        description: Beschreibung
+  activerecord:
+    attributes:
+      administrator:
+        email: E-Mail
+        password: Passwort
+        password_confirmation: Passwort-Wiederholung
+      answer_possibility_group:
+        answer_possibilities: Antwortmöglichkeiten
       question:
-        question_group: Fragengruppe
-        answers: Antworten
         answer_possibilities: Antwortmöglichkeiten
+        answers: Antworten
+        description: Beschreibung
+        question_group:
+          one: Fragengruppe
+          other: Fragengruppen
       survey:
-        active?: Aktiv
-        question_groups: Fragengruppen
+        active: Aktiv
+        active_from: Aktiv von
+        active_to: Aktiv bis
+        inactive: Inaktiv
         title_de: Deutscher Titel
         title_fr: Französischer Titel
         title_it: Italienischer Titel
-        active_from: Aktiv von
-        active_to: Aktiv bis
-  activerecord:
+      survey_entry:
+        answers: Antworten
+        survey: Umfrage
+    errors:
+      models:
+        survey_entry:
+          attributes:
+            answers:
+              not_all_answered: es wurden nicht alle Fragen beanrwortet
     models:
       administrator:
         one: Administrator
         other: Administratoren
-      survey_entry:
-        one: Umfrageneingabe
-        other: Umfrageneingaben
-      survey:
-        one: Umfrage
-        other: Umfragen
       answer:
         one: Antwort
         other: Antworten
       answer_possibility:
         one: Antwortmöglichkeit
         other: Antwortmöglichkeiten
-      question_group:
-        one: Fragengruppe
-        other: Fragengruppen
+      answer_possibility_group:
+        one: Antwortmöglichkeitgruppe
+        other: Antwortmöglichkeitgruppen
       question:
         one: Frage
         other: Fragen
-    attributes:
-      administrator:
-        email: E-Mail
-        password: Passwort
-        password_confirmation: Passwort-Wiederholung
+      question_group:
+        one: Fragengruppe
+        other: Fragengruppen
       survey:
-        title_de: Deutscher Titel
```

Comment:
Why did you delete all those activerecord translations?

Reply:
Not deleted moved. Far down.